# HG changeset patch
# User Benjamin Lindner <lindnerb@users.sourceforge.net>
# Date 1243152125 -7200
#      Sun May 24 10:02:05 2009 +0200
# Node ID a1d20052517a950033896f5a876de8f3ee520138
# Parent  f8b8ab529913fc3291a563223f13fe26842e97f5
support for mkoctfile.exe and octave-config.exe on mingw platform

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,13 @@
+2009-05-24  Benjamin Lindner <lindnerb@users.sourceforge.net>
+
+	* configure.in: (SCRIPTS_EXE_SUFFIX) Define to ".exe" for 
+	mingw32 compilation (similar to msvc)
+
 2009-05-24  Robert T. Short  <octave@phaselockedsystems.com>
 
 	* examples/@FIRfilter: added FIRfilter class example.
 
 2009-05-22  Marco Atzeri  <marco_atzeri@yahoo.it>
 
         * src/sysdep.cc: Removed CYGWIN_init 
 
diff --git a/configure.in b/configure.in
--- a/configure.in
+++ b/configure.in
@@ -1268,16 +1268,17 @@ case "$canonical_host_type" in
     FPICFLAG=
     SHLEXT=dll
     SHLLIB=dll.a
     SHLBIN=dll
     DL_LDFLAGS="-shared -Wl,--export-all-symbols -Wl,--enable-auto-import -Wl,--enable-runtime-pseudo-reloc"
     SH_LDFLAGS="-shared -Wl,--export-all-symbols -Wl,--enable-auto-import -Wl,--enable-auto-image-base"
     SONAME_FLAGS='-Wl,--out-implib=$@.a'
     library_path_var=PATH
+    SCRIPTS_EXE_SUFFIX='$(EXEEXT)'
   ;;
 
   *-*-msdosmsvc)
     DL_LDFLAGS="-shared"
     CPICFLAG=
     CXXPICFLAG=
     FPICFLAG=
     SHLEXT=dll
